telegram/user.py

```diff
@@ -105,3 +111,29 @@ def de_list(cls, data, bot):
             users.append(cls.de_json(user, bot))
 
         return users
+
+    def mention_markdown(self, name=None):
+        """
+        Args:
+            name (:obj:`str`): If provided, will overwrite the user's name.
+
+        Returns:
+            :obj:`str`: The inline mention for the user as markdown.
+        """
+        if not name:
+            return util_mention_markdown(self.id, self.name)
+        else:
+            return util_mention_markdown(self.id, name)
+
+    def mention_html(self, name=None):
+        """
+        Args:
+            name (:obj:`str`): If provided, will overwrite the user's name.
+
+        Returns:
+            :obj:`str`: The inline mention for the user as HTML.
+        """
+        if not name:
+            return util_mention_html(self.name, self.id)
```

Comment:
you are absolutely right. I mixed up with the old versions. thank you for spotting it out. pushing new commit now.